pig/src/main/java/org/jboss/pnc/bacon/pig/Pig.java

```diff
@@ -254,9 +254,17 @@ public String doExecute() {
                 description = SKIP_BRANCH_CHECK_DESC)
         private boolean skipBranchCheck;
 
+        @Option(
```

Comment:
`tempBuild` is already defined in `PigCommand`, I don't think it's needed here

Reply:
Feel free to merge when this is fixed, the rest looks good.
Thank you!

Reply:
Ah thanks for letting me know! I didn't even check! :)